apps/web/src/app/(private)/settings/_components/ProviderApiKeys/New/index.tsx

```diff
@@ -22,6 +23,10 @@ export default function NewApiKey({
     onSuccess: () => setOpen(false),
   })
 
+  const [providerType, setProviderType] = useState(Object.keys(Providers)[0]);
```

Comment:
I know is not part of your change but we already have this defiend in form of array:

```javascript
const PROVIDER_OPTIONS = Object.entries(Providers).map(([key, value]) => ({
  value,
  label: key,
})

function Component () {
  const [providerType, setProviderType] = useState(PROVIDER_OPTIONS[0].value);

  return (
    <Select
     required
     label='Provider'
     name='provider'
     onChange={(newValue) => setProviderType(newValue)}
     defaultValue={data?.provider}
     options={PROVIDER_OPTIONS}
   />
 )
}
```